### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by Hsankesara**
Solving issue #9

### CODE REVIEW

File: model-invocation-endpoint/README.md [hunk 1]
```diff
@@ -2,7 +2,25 @@
...
+
+#### Setting up the ENV file
+
+Create the `.env` file.
+
+```bash
+cp env.template .env
+```
+
+Use the same `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` values that have been used in `mlflow-server`
```
Comment (yatharthranjan) [inline model-invocation-endpoint/README.md:21]: just wondering why it needs access to these? Does it access the S3 storage in some way?

Comment (Hsankesara) [reply model-invocation-endpoint/README.md:21]: Yes, you would need those to access the minio server and fetch saved artifacts.

Comment (yatharthranjan) [reply model-invocation-endpoint/README.md:21]: Ah ok, so it is directly accessing and making changes to the artifact storage. Initially, I thought the mlflow server would expose those in the client api(or rest api) and the invocation endpoint will not need to access the artifact storage directly.

**Review by yatharthranjan — COMMENTED** [review]
LGTM

**Review by yatharthranjan — APPROVED** [review]

**Pull request merged**